integration-tests/smoke/reorg_above_finality_test.go

```diff
@@ -15,25 +16,39 @@ import (
 	tc "github.com/smartcontractkit/chainlink/integration-tests/testconfig"
 )
 
-func TestReorgAboveFinality(t *testing.T) {
+func TestReorgAboveFinality_FinalityTagDisabled(t *testing.T) {
 	t.Parallel()
 
 	l := logging.GetTestLogger(t)
-	config, err := tc.GetConfig("Smoke", tc.OCR2)
+	config, err := tc.GetConfig(t.Name(), tc.LogPoller)
 	require.NoError(t, err, "Error getting config")
 
 	privateNetworkConf, err := actions.EthereumNetworkConfigFromConfig(l, &config)
 	require.NoError(t, err)
 
-	nodeFinalityDepthInt := int64(10)
+	// Get values from the node config
+	configMap := make(map[string]interface{})
+	err = toml.Unmarshal([]byte(config.NodeConfig.CommonChainConfigTOML), &configMap)
+	if err != nil {
+		log.Fatalf("Error unmarshaling TOML: %s", err)
+	}
+	nodeFinalityDepthInt, isFinalityDepthSet := configMap["FinalityDepth"].(int64)
+	nodeFinalityTagEnabled := configMap["FinalityTagEnabled"].(bool)
```

Comment:
sure you don't wanna do it in a more controlled manner? maybe using that function I shared with you? this can result in 2 panics:
* nil key in the map
* different than expected value type

Reply:
@Tofel yeah, I know this. But do we really need this in tests? Even if it panics it's veeery easy to understand why it did.

Reply:
up to you